### ISSUE TITLE
If possible, can you use the Software Center interface abandoned by Ubuntu as the UI environment of AppShop 

### ISSUE THREAD

**Issue by louies0623** [linuxkettle/AppShop#8]
Maybe I am still immersed in the beauty of the pre-Unity era, but this interface may be able to be applied to the UI of this project .
Just include Progress in Installed and add updates .

![image](https://user-images.githubusercontent.com/44593430/124320957-6bd7da00-dbaf-11eb-84e1-32ebe7e934e6.png)

![image](https://user-images.githubusercontent.com/44593430/124321026-8ca02f80-dbaf-11eb-9aa1-fe53fabf4adf.png)

![image](https://user-images.githubusercontent.com/44593430/124321124-c07b5500-dbaf-11eb-9832-507c126a9162.png)

![image](https://user-images.githubusercontent.com/44593430/124321380-2962cd00-dbb0-11eb-8136-8b8d3ba2e952.png)

![image](https://user-images.githubusercontent.com/44593430/124321438-40092400-dbb0-11eb-9477-915e73d4fe14.png)

**Comment by kettle-7** [comment]
@probonopd has made it clear he's not interested in an app store. I am developing a third-party solution if you want it, but it's not allowed to be attached to helloSystem.

**Comment by jhjacobs81** [comment]
It could still he included/installed on linux, or even on helloSystem afterwards?

**Comment by kettle-7** [comment]
Oh, sorry, I thought this issue was on helloSystem repo!

**Comment by jhjacobs81** [comment]
Check Matrix ;-)

**Comment by kettle-7** [comment]
> add updates

That will take a while. helloSystem updates *could* be done, but for Linux I'd need to include update logic for every distro that exists!

Going on there now shows "Coming soon!"

> include Progress in Installed

The good thing here is that the apps don't get installed, they're just downloaded.

**Comment by ghost** [comment]
Couldn't updates be done though AppImageUpdate?

**Comment by kettle-7** [comment]
You can use that for specific AppImages that implement it, you need like a manifest thingy.

System updates would only be done on our debian based distro and maybe helloSystem.

**Comment by ghost** [comment]
I feel like system updates should not be a priority as Apps are the priority. 
Just a thought.

**Comment by jhjacobs81** [comment]
and why is that? Please enlighten us :)

**Comment by ghost** [comment]
The priority of the project in my mind are apps, as the name suggests.
System updates can come at a later time.

**Comment by jhjacobs81** [comment]
Fair enough :)

**Comment by kettle-7** [comment]
Oh yeah, we haven't published much about Omega yet 🙃

**Comment by jhjacobs81** [comment]
;-) im still making a product info thingy :)

**Comment by kettle-7** [comment]
But system updates will be shown on Oslomega

**Comment by jhjacobs81** [comment]
Yes, but lets not talk about that just yet :) at least not untill we have a site up for people to actually see :-)

**Comment by ghost** [comment]
So we're not following this GUI implementation but rather our own
